README.md

```diff
@@ -237,6 +237,13 @@ custom:
       custom_domains:
         - my-container.some.domain.com
 
+      # Health check configuration
+      healthCheck:
```

Comment:
For reviewers, this isn't quite the same as the REST API, but I think providing semi-empty objects doesn't work well in YAML. However, the argument are named the same.